### ISSUE TITLE
Validation of `protonmail` addresses have 3x latency

### ISSUE THREAD

**Issue by cortopy** [reacherhq/check-if-email-exists#1181]
Using the Reacher api, a validation of a `protonmail` address takes around 7.20-7.50 seconds. I find this latency a bit excessive, especially considering a gmail address takes less than 2.5 seconds.

I'm not sure if this is `protonmail` slowing requests or there is something that could be done configuration-wise

**Comment by Bayernatoor** [comment]
Yep agreed, I am thinking it may be protonmail but verifications of `protonmail` addresses can take > 10 seconds on my end.
